src/arm-codegen.c

```diff
@@ -1,532 +1,457 @@
 /* Translate IR to target machine code */
 
 #include "arm.c"
+#include "reg-alloc.c"
```

Comment:
Instead of the inclusion of `reg-alloc.c`, can you specify the header?

Reply:
Is it better that moving `dump_ir()`and the inclusion of `reg-alloc.c` into `main.c`?